tests/test_hwm_pubsub.cpp

```diff
@@ -259,7 +259,7 @@ void test_blocking (const char *bind_endpoint_)
 DEFINE_REGULAR_TEST_CASES (tcp, "tcp://127.0.0.1:*")
 DEFINE_REGULAR_TEST_CASES (inproc, "inproc://a")
 
-#if !defined(ZMQ_HAVE_WINDOWS) && !defined(ZMQ_HAVE_GNU)
+#if !defined(ZMQ_HAVE_GNU)
```

Comment:
ipc is not available everywhere on windows, this is wrong

Reply:
It's better to have the test fail where it's not supported than not testing where it is.